spec/features/activity_pages/activity_pages_spec.rb

```diff
@@ -75,6 +95,49 @@
           end
         end
       end
+
+      describe 'filters' do
```

Comment:
Can we also add a few cases to catch passing bad params (e.g. invalid date, invalid user, invalid type)?